C/Cpp_include/c4BlobStore.hh

```diff
@@ -106,15 +107,15 @@ struct C4WriteStream
     : public fleece::InstanceCounted
     , C4Base {
     explicit C4WriteStream(C4BlobStore&);
-    C4WriteStream(C4WriteStream&&);
-    ~C4WriteStream();
+    C4WriteStream(C4WriteStream&&) noexcept;
+    ~C4WriteStream() override;
 
-    C4BlobStore& blobStore() const { return _store; }
+    [[nodiscard]] C4BlobStore& blobStore() const { return _store; }
```

Comment:
I believe Clang-Tidy recommends marking method as `nodiscard` if a method is returning a reference or pointer, because not using the return value from such a method is usually a programming error.